### PR TITLE
fix: Update shutter logo

### PR DESCRIPTION
**Pull request by ChaituVR**
### Summary

- Update shutter logo with new one from https://www.shutter.network/
- Remove `fill="white"` on "shutter" keyword to make it work on light mode
- Remove "powered by" and make shutter logo look same everywhere

Active Proposal:
| Before    | After |
| -------- | ------- |
| Untitled 5  | Untitled 6    |
| Untitled 8 |  Untitled 7 |

Closed Proposal:
| Before    | After |
| -------- | ------- |
| Untitled 11 | Untitled 12 |
|  Untitled 9  | Untitled 10  |

Closes https://github.com/snapshot-labs/workflow/issues/136

### How to test

1. Go to proposals:
http://localhost:8080/#/s:shutterdao0x36.eth/proposal/0xfbb3c3a53830903896bac3c74eab2c5b60909dabc3cc01ff2a4c08a27f262ded
http://localhost:8080/#/s:shutterdao0x36.eth/proposal/0x4bb5cf27998b7371d1c69f7d2d0e433d601bcb46e2a2cdd38108a743c432fa35
3. You should see an updated logo

### CODE REVIEW

**Comment by bonustrack** [comment]
The logo should use only a single color, white and black (using link color) and without blue

**Comment by bonustrack** [comment]
No

**Review by wa0x6e — APPROVED** [review]
tAck

File: apps/ui/src/components/ProposalResults.vue [hunk 1]
```diff
@@ -93,12 +93,12 @@ const otherResultsSummary = computed(() => {
     <div>
       <a
         href="https://blog.shutter.network/announcing-shutter-governance-shielded-voting-for-daos/"
+        class="text-skin-text"
```
Comment (wa0x6e) [inline apps/ui/src/components/ProposalResults.vue:96]: We could add `flex items-center` here, and remove the `inline-block` on both child elements

**Pull request merged**